site/src/pages/WorkspacePage/WorkspaceScheduleControls.tsx

```diff
@@ -79,161 +97,40 @@ export const WorkspaceScheduleControls: FC<WorkspaceScheduleControlsProps> = ({
       )}
 
       {canUpdateSchedule && canEditDeadline(workspace) && (
-        <span css={styles.scheduleControls}>
-          <Popover>
-            <PopoverTrigger>
-              <IconButton
-                disabled={!deadlineMinusEnabled}
-                size="small"
-                title="Subtract hours from deadline"
-                css={styles.scheduleButton}
-              >
-                <RemoveIcon />
-              </IconButton>
-            </PopoverTrigger>
-            <PopoverContent
-              id="schedule-sub"
-              classes={{ paper: classNames.paper }}
-              horizontal="right"
+        <div css={styles.scheduleControls}>
+          <Tooltip title="Subtract 1 hour from deadline">
+            <IconButton
+              aria-label="Subtract hour"
```

Comment:
Could you replace this with visually-hidden text in the button itself? (Same for the add button)

Reply:
Sure thing